include/deal.II/matrix_free/matrix_free.h

```diff
@@ -1892,8 +1892,14 @@ class MatrixFree : public Subscriptor
   /**
    * Return the category the current batch range of cells was assigned to.
    * Categories run between the given values in the field
-   * AdditionalData::cell_vectorization_category for non-hp-DoFHandler types
+   * AdditionalData::cell_vectorization_category for the non-hp case
```

Comment:
@kronbichler There have been other places with references to `non-hp-DoFHandler` ;)